### ISSUE TITLE
Option to require nesting selectors

### ISSUE THREAD

**Issue by nschonni** [brigade/scss-lint#107]
Although some people prefer not to nest selectors, it would be nice to have an option to enforce the nesting.

**Bad**:

``` scss
.foo .bar {
   color: #fff;
}
```

**Good**:

``` scss
.foo {
  .bar {
     color: #fff;
  }
}
```
